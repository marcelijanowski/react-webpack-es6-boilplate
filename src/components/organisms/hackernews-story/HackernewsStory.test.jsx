import React, { PureComponent } from 'react'
import renderer from 'react-test-renderer';

import HackerNewsStory from './HackernewsStory';

describe('HackerNewsStory container', () => {
  it('renders a snapshot when not props are set', () => {
    const item = {
      id: 1
    }
    const tree = renderer.create(<HackerNewsStory item={item}/>).toJSON();
    expect(tree).toMatchSnapshot();
  });
  it('render a snapshot when props are set and page is loading', () => {
    const item = {
      id: 1
    }
    const tree = renderer.create(<HackerNewsStory item={item} />);
  });
})