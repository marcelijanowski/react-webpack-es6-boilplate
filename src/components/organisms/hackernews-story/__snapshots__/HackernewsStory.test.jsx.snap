// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HackerNewsStory container renders a snapshot when not props are set 1`] = `
<li
  class="mdc-list-item"
>
  <span
    class="mdc-list-item__text"
  >
    1
    <span
      class="mdc-list-item__secondary-text"
    >
      Second-line text
    </span>
  </span>
</li>
`;
